packages/phone-number-privacy/signer/README.md

```diff
@@ -6,21 +6,38 @@ A service run by distributed partners that generates unique partial signatures f
 
 You can use the following environment variables to configure the Phone Number Privacy - Signer service:
 
-- `NODE_ENV` - eg. development or production
+- `NODE_ENV` - `development` or `production`
 - `SERVER_PORT` - The port on which the express node app runs (8080 by default).
-- `BLOCKCHAIN_PROVIDER` - The blockchain node provider for on chain requests. This could be a node with RPC set up.  For development testing you can use https://alfajores-forno.celo-testnet.org
-- `DB_HOST` - The URL under which your database is accessible.  Supports Postgres, MSSQL, MySQL, MariaDB, SQLite3, Oracle.
+- `BLOCKCHAIN_PROVIDER` - The blockchain node provider for chain state access. This could be a node with RPC set up.  For development testing you can use `https://alfajores-forno.celo-testnet.org`. For Mainnet, it would be `https://rc1-forno.celo-testnet.org`
```

Comment:
do we want to encourage the user of forno.  Maybe make this softer offer (as most individuals we chatted with seemed to think it was counter productive to use the centralized forno endpoint).

Maybe `If you chose not to run your own node with this service, you can opt to use the `https://rc1-forno.celo-testnet.org`

Reply:
Good idea, I'll change it